app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h2>Sign up</h2>
```

Comment:
- [ ] An error (CSRF token) occurred when signing into the app, kindly fix it so you can log in

![Screenshot_20220928_010312](https://user-images.githubusercontent.com/25227022/192658468-eb15112e-1350-4d59-b087-5c8b32975a6a.png)

Reply:
I haven't got this problem, but I've seen people have it when they use older versions of Ruby